### PR TITLE
moving workload to mirrors-repo

### PR DESCRIPTION
**Pull request by Gregory-Pereira**
/cc @lmilbaum + @sallyom , moving this workload to the mirrors-repo.

### CODE REVIEW

**Review by lmilbaum — CHANGES_REQUESTED** [review]
Should this fix the login triggering issue? If not, what does it solve?

**Comment by Gregory-Pereira** [comment]
This was the only difference between this workflow and our model-servers workflow which seems to have no trouble authenticating against the registry and pushing to it. I would assume the registry service is required to interact with the registry, and GitHub understood that and prevented it from running. Although I will need to put some thought into our Podman filesystem usage now it looks like ...

**Comment by lmilbaum** [comment]
> This was the only difference between this workflow and our model-servers workflow which seems to have no trouble authenticating against the registry and pushing to it. I would assume the registry service is required to interact with the registry, and GitHub understood that and prevented it from running. Although I will need to put some thought into our Podman filesystem usage now it looks like ...

I think the service is required for sharing locally built image by multiple jobs. Not sure if that is needed in this workflow.

File: .github/workflows/instructlab_baseimages_build_push.yaml [hunk 1]
```diff
@@ -25,15 +25,15 @@ env:
...
+    if: "!contains(github.event.pull_request.labels.*.name, 'hold-tests') && github.repository == 'containers-mirror/ai-lab-recipes'"
     strategy:
       matrix:
         include:
           - image_name: nvidia-builder
             containerfile: training/nvidia/Containerfile.builder
             context: training/nvidia
             platforms: linux/amd64
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```
Comment (lmilbaum) [inline .github/workflows/instructlab_baseimages_build_push.yaml]: Is this change required for the scope of this PR? I actually don't think it is a good idea to use ephemeral images.
Please update the PR description to reflect the change.

**Review by lmilbaum — CHANGES_REQUESTED** [review]
